### PR TITLE
fix: update fields of 132

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Update the rfc 132 - merge mining rfc, some fields were outdated.

### CODE REVIEW

**Comment by SWvheerden** [comment]
needs a change log as well.

**Comment by SWvheerden** [comment]
ACK

File: src/RFC-0132_Merge_Mining_Monero.md [hunk 1]
```diff
@@ -173,6 +173,12 @@ so that the miner submits the found block at either chain's difficulty. The Tari
 The miner submits a solved Monero block (at a difficulty of `min(monero_difficulty, tari_difficulty)`) to the proxy. The cached
 Tari block is retrieved, enriched with the `MoneroPowData` struct and submitted to the Tari base node.
 
+# Change Log
+
+| Date        | Change              | Author    |
+|:------------|:--------------------|:----------|
+| 21 Oct 2022 | Update fields       | Cifko     |
```
Comment (CjS77) [inline src/RFC-0132_Merge_Mining_Monero.md:180]:
```suggestion
| 26 Oct 2022 | Stablise RFC       | CjS77    |
| 21 Oct 2022 | Update fields       | Cifko     |
```

**Review by CjS77 — COMMENTED** [review]
There are a few LaTex formulae that aren't rendering properly, but otherwise LGTM

**Review by CjS77 — APPROVED** [review]
Stabilised

**Pull request merged**